xml/System/String.xml

```diff
@@ -9685,10 +9695,13 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
    
   
 ## Examples  
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
  The following example demonstrates the <xref:System.String.Length%2A> property.  
   
  [!code-cpp[string.length#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp#1)]
- [!code-csharp[string.length#1](~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs#1)]
+ [!code-csharp-interactive[string.length#1](~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs#1)]
```

Comment:
the example from the remarks should also be interactive

Reply:
Done in https://github.com/dotnet/samples/pull/1171